cypress/pages/LandingPage.js

```diff
@@ -12,6 +12,10 @@ class LandingPage{
     return cy.get('.active > .product-select__text').click()
   }
 
+  selectProducts(){
+    return cy.get('.active > .product-select__text').click()
```

Comment:
What is the purpose of this lines

Reply:
I think its duplicate code. Let me remove.

Reply:
I have fixed it. please review again